env_spec.py

```diff
@@ -0,0 +1,90 @@
+spec_str = 'DATABASE_URL\nADMIN_EMAIL\nDEBUG'
+
+
+def str_split(spec_str):
+    """
+    split input string by \n and returns strings into an array
+    """
+    field_array = []
+    field_array = spec_str.split('\n')
+    
+    return field_array
+
+def check_latin_chars(input_str):
+    """
+    function that checks if input_str consists of latin chars 
+    (input_str can contain underscore and numeric digits)
+    """
+
+    try:
+        input_str.encode('utf-8').decode('ascii')
+    except Exception:
+        return -1
+    else:
+        return 0
+
+
+def check_symbols(input_str):
+    """
+    checks every char of input_str (must be digit/underscore/uppercase latin char)
+    return -1 for error about specifications or 0 if correct
+    """
+
+    for i in input_str:   
+        #given specifications
+        if(i.isdigit() == True or i == '_' or (i >= 'A' and i <= 'Z')):
+            continue
+        else:
+            return -1
+        
+    return 0
+
+
+def check_input(field_array):
```

Comment:
You can drastically simplify input checking by using the [`re` Python module](https://docs.python.org/3/library/re).

I would use [`re.match`](https://docs.python.org/3/library/re.html#re.match) to match every line with the following Regular Expression:

```python
alphanumeric_that_does_not_start_with_digit = r'^[A-Z_][0-9A-Z_]*$';
```

Reply:
Also, instead of returning `0` and `-1`, I would go with `True` and `False`, as they are a bit more humane 😄.